lib/README.md

```diff
@@ -0,0 +1,6 @@
+How to use chat room:
```

Comment:
@loganlsr Good job adding a readme!  Going forward, start to use additional markdown components and add more layers for a better layout and to structure your documentation to conform to that of other industry documentation standards